kpatch-build/create-diff-object.c

```diff
@@ -788,6 +838,84 @@ static bool kpatch_line_macro_change_only(struct kpatch_elf *kelf,
 	return true;
 }
 
+static bool _kpatch_line_macro_change_only_aarch64(struct kpatch_elf *kelf,
```

Comment:
Do we really need an arm64-specific version?  At least from a quick glance I don't see why the generic version wouldn't work.